Packs/Campaign/Scripts/SplitCampaignContext/SplitCampaignContext.py

```diff
@@ -0,0 +1,76 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from typing import Tuple
+
+BELOW_THRESHOLD_ITEMS_CONTEXT_PATH = 'LowerSimilarityIncidents'
+ABOVE_THE_THRESHOLD_ITEMS_CONTEXT_PATH = 'incidents'
+
+
+def save_to_context(items: list, context_path: str, delete_existing: bool = False, is_sub_playbook: str = 'auto',
+                    table_header='Incidents Result'):
+    if delete_existing:
+        res = demisto.executeCommand('DeleteContext', {"key": context_path, "subplaybook": is_sub_playbook})
+        if is_error(res):
+            return_error('Failed to delete current context. Error details:\n{}'.format(get_error(res)))
```

Comment:
no need to change this :) but for next time use CommonServerPython's `execute_command()`